assets/scripts/main.js

```diff
@@ -86,7 +86,7 @@
   $(document).ready(() => {
     // Disable copy/paste actions on billing email fields.
     if (shop_standards_settings.emailConfirmationEmail === 'yes') {
```

Comment:
Wny didn't you pass in the parameter `shop_standards_settings` as well, if you did it for the `admin.js` file?

Reply:
I only fixed all the "errors" that the linter was showing, but you're totally right.